CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # package:sentry changelog
 
+## 2.4.0
```

Comment:
How about making it a `3.0.0-dev0`, given how big and probably subtly breaking this change is?